OWML.ModHelper.Menus/ModSliderInput.cs

```diff
@@ -1,27 +1,31 @@
 ﻿using OWML.Common.Menus;
 using OWML.ModHelper.Events;
 using UnityEngine;
+using UnityEngine.UI;
 
 namespace OWML.ModHelper.Menus
 {
     public class ModSliderInput : ModInput<float>, IModSliderInput
     {
         public float Min { get; set; }
         public float Max { get; set; }
+        public bool HasValueText => _valueText != null;
```

Comment:
Is this needed?

Reply:
Check this line: https://github.com/amazingalek/owml/pull/224/files#diff-a4fa79456d54d033ef6fe8039ce95a34R97

I use this so that the template slider element is one that has a value being displayed. For instance, FOV slider would be a good template (because it shows the value), but Gamma slider wouldn't.

FOV is the first slider in the graphics tab so getting the first slider would still work, but if they add a new slider or change the order in an update it could break.